app/src/main/java/org/oppia/app/player/audio/CellularDataInterface.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.player.audio
+
+/** Interface to check the preference regarding alert for [CellularDataDialogFragment] */
+interface CellularDataInterface {
+  fun cellularDataUserPreference(doNotShowAlert: Boolean, playAudio: Boolean)
```

Comment:
For perhaps a simpler interface, consider doubling the method based on the play boolean state:

```kotlin
enableAudioWhileOnCellular(doNotShowAgain: Boolean)
disableAudioWhileOnCellular(doNotShowAgain: Boolean)
```

Reply:
Changed the interface as per suggestions.